redirects.yaml

```diff
@@ -326,6 +326,8 @@ redirects:
     destination: https://developer.chrome.com/docs/web-platform/handwriting-recognition/
   - source: /virtualkeyboard
     destination: https://developer.chrome.com/docs/web-platform/virtual-keyboard/
+  - source: /workingoffthegrid
+  - destination: https://developer.chrome.com/docs/workbox/service-worker-overview/
```

Comment:
I think this is a bug, and dash shouldn't be there @malchata . Will open a follow up PR.